products.js

```diff
@@ -227,15 +228,15 @@ class DiffStats {
         if (this.feature === 'image') {
           //compare images by src
           expectedText = expectedDom.body.firstChild.src;
-          gotText = this.tuningRoutine(sourceDom, dict).map(fnode => fnode.element.src)[0];
+          gotText = this.tuningRoutine()(sourceDom).map(fnode => fnode.element.src)[0];
```

Comment:
Hm I'm not quite sure what you mean, the dictionary depends on the folder,  so I don't think I could bundle it in with the feature - tuning routine map. I've also changed this code slightly, will push changes to a new branch soon